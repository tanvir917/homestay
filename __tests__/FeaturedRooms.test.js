import React from 'react'
import TestRenderer from 'react-test-renderer';
import FeaturedRooms from '../src/Components/FeaturedRooms/FeaturedRooms';


it('renders correctly', () => {
  const tree = TestRenderer
    .create(<FeaturedRooms />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});