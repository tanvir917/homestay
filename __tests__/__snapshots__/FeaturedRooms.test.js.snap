// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="featured-rooms"
>
  <div
    className="section-title"
  >
    <h4>
      featured rooms
    </h4>
    <div />
  </div>
  <div
    className="featured-rooms-center"
  >
    <div
      className="loading"
    >
      <h4>
        rooms data loading ...
      </h4>
      <img
        alt="loading gif"
        src={
          Object {
            "process": [Function],
          }
        }
      />
    </div>
  </div>
</section>
`;
